GitExtensions/GitExtensions.csproj

```diff
@@ -7,6 +7,12 @@
     <!-- Default debugging command starts GitExtensions browsing this repository -->
     <StartArguments>browse "$(MSBuildThisFileDirectory).."</StartArguments>    
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <Prefer32bit>true</Prefer32bit>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+    <Prefer32bit>true</Prefer32bit>
+  </PropertyGroup>
```

Comment:
Is there a reason we can't add to the above `PropertyGroup`: `<Prefer32bit>true</Prefer32bit>`?